build.psm1

```diff
@@ -447,20 +447,24 @@ function Start-PSBuild {
     }
 
     function Stop-DevPowerShell {
+        if (-not (Test-Path $script:Options.Output)) {
+            return
```

Comment:
This return path needs to write warning/error.

Reply:
Yes. How about
```PowerShell
Write-Warning "Could not find dev powershell at '$($script:Options.Output)'"
```

Reply:
Added warning.